js/shopping-list-item.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+class ShoppingListItem {
+  constructor(name, description) {
+    this.name = name;
+    this.description = description;
+    this.isDone = false;
+  }
+
+  check() {
+    this.isDone = true;
+  }
+
+  uncheck() {
+    this.isDone = false;
+  }
+
+  render(idx) {
+    let isChecked = '';
+    let strikeStart = '';
```

Comment:
Instead of using the strike element, you can achieve the same effect using a css rule: https://css-tricks.com/almanac/properties/t/text-decoration/